### PR TITLE
dxpy 0.298.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/dxpy.rb [hunk 1]
```diff
@@ -50,6 +50,11 @@ class Dxpy < Formula
     sha256 "a0c30272fb4ddda5f5ffc1089d7405b7a71b0b0f51993cb4e5dbb4590b2fc229"
   end
 
+  resource "gnureadline" do
+    url "https://files.pythonhosted.org/packages/50/64/86085c823cd78f9df9d8e33dce0baa71618016f8860460b82cf6610e1eb3/gnureadline-6.3.8.tar.gz"
+    sha256 "58e208ed34a53456ecf58bc11eacbe332c50d322b329457e2c645bd15192b035"
+  end
```
Comment (chenrui333) [inline Formula/dxpy.rb:56]: this is not needed.

Comment (Rylan12) [reply Formula/dxpy.rb]: Why not? It's included in [`setup.py`](https://github.com/dnanexus/dx-toolkit/blob/d22239c6dddad2004cdc346ef2ba99840a7fc73d/src/python/requirements.txt#L9)

Comment (chenrui333) [reply Formula/dxpy.rb]: because it is caused build issue, and I did include `readline` as separate dependency.

Comment (chenrui333) [reply Formula/dxpy.rb]: looks like we have build auto-cancelled now, too bad that I could show what I said. I reset the commit to prove the point.

Comment (Rylan12) [reply Formula/dxpy.rb:56]: That's fine (no need to prove it I was just curious)

Comment (Rylan12) [reply Formula/dxpy.rb:56]: Can we put a comment somewhere to indicate that `gnureadline` shouldn't be added? `brew bump-formula-pr` will probably add it each time.

Comment (chenrui333) [reply Formula/dxpy.rb:56]: Sounds good, always to have this knowledge somewhere is good :)

Comment (chenrui333) [reply Formula/dxpy.rb:56]: adding a comment now.

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**